collection-de-l-ecole-francaise-de-rome-full-note.csl

```diff
@@ -0,0 +1,319 @@
+          <text term="no date" form="short"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="edition">
+      <choose>
+        <if is-numeric="edition">
+          <group delimiter=" ">
+            <number variable="edition" form="ordinal"/>
+            <text term="edition" form="short" text-case="lowercase"/>
+          </group>
+        </if>
+        <else>
+          <text variable="edition"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="thesis">
+      <group prefix=" " delimiter=", ">
+        <text macro="title" font-style="italic"/>
+        <text variable="genre"/>
+        <text variable="publisher"/>
+        <text macro="year-date"/>
+      </group>
+    </macro>
+    <macro name="articles">
+      <group delimiter=", ">
+        <text macro="title" quotes="true"/>
+        <group delimiter=" ">
+          <text term="in"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+        <text variable="volume"/>
+        <text variable="issue"/>
+        <group delimiter=" ">
+          <text macro="year-date"/>
+          <group prefix="(" suffix=")" delimiter=", ">
+            <text variable="URL"/>
+            <text macro="access"/>
+          </group>
+        </group>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <macro name="book">
+      <group delimiter=", ">
+        <text macro="title"/>
+        <text macro="edition"/>
+        <text macro="publisher"/>
+        <group delimiter=" ">
+          <text macro="year-date"/>
+          <text macro="collection"/>
+        </group>
+      </group>
+    </macro>
+    <macro name="dictionary">
+      <group delimiter=", ">
+        <text macro="title" quotes="true"/>
+        <group delimiter=" ">
+          <text term="in"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+        <group delimiter=" ">
+          <label variable="volume" form="short"/>
+          <text variable="volume"/>
+        </group>
+        <text macro="year-date"/>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <macro name="chapter">
+      <group delimiter=" ">
+        <text macro="title" quotes="true" suffix=","/>
+        <text term="in"/>
+        <text macro="editor" suffix=","/>
+        <text variable="container-title" font-style="italic" suffix=","/>
+        <text macro="publisher" suffix=","/>
+        <group delimiter=" " suffix=",">
+          <text macro="year-date"/>
+          <text macro="collection"/>
+        </group>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <citation disambiguate-add-year-suffix="true" et-al-subsequent-min="2" et-al-subsequent-use-first="1">
+      <layout delimiter=" ; ">
+        <choose>
+          <if position="subsequent">
+            <group delimiter=", ">
+              <text macro="author"/>
+              <text variable="title" form="short" font-style="italic" suffix="..."/>
```

Comment:
I tried with the ASCII code &#133; for the ellipsis but obtain a blank 
square in Word, so I got back to three dots
What would be the correct code?

Le 07/12/2015 20:18, Sebastian Karcher a écrit :
>
> In collection-de-l-ecole-francaise-de-rome-full-note.csl 
> <https://github.com/citation-style-language/styles/pull/1805#discussion_r46863177>:
>
> > +        <text variable="container-title" font-style="italic" suffix=","/>
> > +        <text macro="publisher" suffix=","/>
> > +        <group delimiter=" " suffix=",">
> > +          <text macro="year-date"/>
> > +          <text macro="collection"/>
> > +        </group>
> > +        <text macro="pages"/>
> > +      </group>
> > +    </macro>
> > +    <citation disambiguate-add-year-suffix="true" et-al-subsequent-min="2" et-al-subsequent-use-first="1">
> > +      <layout delimiter=" ; ">
> > +        <choose>
> > +          <if position="subsequent">
> > +            <group delimiter=", ">
> > +              <text macro="author"/>
> > +              <text variable="title" form="short" font-style="italic" suffix="..."/>
>
> you actually mean to put this here? If so, I'd use the ellipsis 
> character rather than three single periods, i.e. … rather than ... 
> (looks the same, but try selecting. Has the advantage of being 
> non-breakable).
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/citation-style-language/styles/pull/1805/files#r46863177>.
>